### DISCUSSION TITLE
archives/2013-07-27-%E5%BE%80%E4%BA%8B%E5%A6%82%E7%83%9F/

### DISCUSSION THREAD

**Discussion by giscus** [williamlfang/williamlfang.github.io#5] [Announcements]
# archives/2013-07-27-%E5%BE%80%E4%BA%8B%E5%A6%82%E7%83%9F/

闲来无事，独自坐在阳台上，夏日的夜晚，却是没风最来得受罪。只是这样闷热的天气，自己的大脑也早已狂躁。一整天在背单词，终日满脑子的符号。那些牛鬼生蛇，那些漫天星宇，那些蛛丝马迹，那些朝花夕拾，那些风霜雪月，那些年少无知，那些异想天开，都在这小小的大脑里，搅拌的晕乎，缠绕着酸楚。

https://williamlfang.github.io/archives/2013-07-27-%E5%BE%80%E4%BA%8B%E5%A6%82%E7%83%9F/

**Comment by 2060490154** [comment]
往事如烟
